pkgs/applications/networking/mailreaders/alot/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, pkgs, pythonPackages
+, withManpage ? true }:
+
+with pythonPackages;
+
+buildPythonApplication rec {
+    rev = "0.6";
+    name = "alot-${rev}";
+    outputs = [ "out"  "man" ];
+
+    disabled = isPy3k;
+
+    src = pkgs.fetchFromGitHub {
+      owner = "pazz";
+      repo = "alot";
+      inherit rev;
+      sha256 = "1x6in9y823zi0k6xk62nwjybkc3pp5n2ml79x4v9jy6fz2klf2mi";
+    };
+
+    postPatch = ''
+      substituteInPlace alot/defaults/alot.rc.spec \
+        --replace "themes_dir = string(default=None)" \
+                  "themes_dir = string(default='$out/share/themes')"
+    '';
+
+    checkInputs =  [ mock pkgs.gnupg ];
+    nativeBuildInputs = stdenv.lib.optionals withManpage [ sphinx ];
+
+    propagatedBuildInputs = [
+        notmuch
+        urwid
+        urwidtrees
+        twisted
+        python_magic
+        configobj
+        gpg
+      ];
+
+    doCheck = false;
+    postBuild= stdenv.lib.optionalString withManpage "make -C docs man";
+
+    postInstall = ''
+      mkdir -p $out/share/applications $out/man
+      cp -r docs/build/man $out/man
+      cp -r extra/themes $out/share
+
+      sed "s,/usr/bin,$out/bin,g" extra/alot.desktop > $out/share/applications/alot.desktop
+      wrapProgram $out/bin/alot \
+        --prefix LD_LIBRARY_PATH : '${pkgs.lib.makeLibraryPath [ pkgs.notmuch pkgs.file pkgs.gpgme ]}'
```

Comment:
Can you not patch the sources to hardcode the paths instead?

Reply:
why ? this is a part I haven't touched so I would rather not change it as it works